recipes/semver200/all/conanfile.py

```diff
@@ -0,0 +1,79 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import apply_conandata_patches, copy, get
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+import os
+
+
+required_conan_version = ">=1.50.0"
+
+
+class SemVer200Conan(ConanFile):
+    name = "semver200"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/zmarko/semver"
+    description = "Semantic versioning for cpp14"
+    topics = ("versioning", "semver", "semantic")
+    settings = "os", "compiler", "arch", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def export_sources(self):
+        for p in self.conan_data.get("patches", {}).get(self.version, []):
+            copy(self, p["patch_file"], self.recipe_folder, self.export_sources_folder)
+
+    def validate(self):
+        if self.info.settings.os == "Windows" and self.info.options.shared:
+            raise ConanInvalidConfiguration("Shared library on Windows is not supported.")
+        if self.info.settings.compiler.cppstd:
+            check_min_cppstd(self, 14)
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True, destination=self.source_folder)
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def generate(self):
+        gt = CMakeToolchain(self)
+        gt.generate()
+
+    def build(self):
+        apply_conandata_patches(self)
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def package(self):
+        copy(self, "LICENSE", src=self.source_folder, dst=os.path.join(self.package_folder, "licenses"))
+        # Parent Build system does not support installation; so we must manually package
+        hdr_src = os.path.join(self.source_folder, "include")
+        hdr_dst = os.path.join(self.package_folder, "include", self.name)
```

Comment:
The project does not have a subdir (as much as thats recommend these days) https://github.com/zmarko/semver/tree/master/include 

We try to accurate package the project. the examples all use `#include "semver200.h"`

```suggestion
        hdr_dst = os.path.join(self.package_folder, "include")
```

Reply:
Packaging `version.h` outside of a subdir i think would be likely to clash no?

Reply:
@szmyd Yes, but we follow the upstream ... It's a bad approach, but it was the author's decision.